tests/test_units/test_objectdataprovider_class.py

```diff
@@ -4,6 +4,13 @@
 from fmu.dataio._definitions import _ValidFormats
 from fmu.dataio._objectdata_provider import ConfigurationError, _ObjectDataProvider
 
+try:
+    import pyarrow  # noqa # pylint: disable=unused-import
+except ImportError:
+    HAS_PYARROW = False
+else:
+    HAS_PYARROW = True
+
```

Comment:
If pyarrow becomes more important, should we have it as a strict dependency?

Reply:
Could be - are there any contexts where PYARROW is not available? I guess we are still in Py36 in some contexts?